assets/dev/js/editor/utils/controls-css-parser.js

```diff
@@ -37,6 +37,33 @@ ControlsCSSParser = elementorModules.ViewModule.extend( {
 		} );
 	},
 
+	updateBackgroundImagesSizes( control, values ) {
+		if ( control.name.includes( 'background_image' ) ) {
+			const imageSize = values.background_dimension || false;
+			if ( imageSize ) {
+				let responsiveSize = 'background_dimension';
+				const breakpoints = elementorFrontend.config.responsive.breakpoints;
+				Object.entries( breakpoints ).forEach( ( [ breakpointName ] ) => {
+					if ( control.name.includes( breakpointName ) ) {
+						responsiveSize += '_' + breakpointName;
+					}
+				} );
+				const sizeValue = values[ responsiveSize ];
```

Comment:
maybe it could just be something like this?
```suggestion
				const sizeValue = values[ sizeControlName + activeBreakpoint ];
```